src/cdk/drag-drop/directives/drag.ts

```diff
@@ -139,6 +141,9 @@ export class CdkDrag<T = any> implements AfterViewInit, OnChanges, OnDestroy {
   /** Class to be added to the preview element. */
   @Input('cdkDragPreviewClass') previewClass: string | string[];
 
+  /** Target for drag clone to append into */
+  @Input('cdkDragCloneTarget') cloneTarget: string | ElementRef<HTMLElement> | HTMLElement;
+
```

Comment:
We may also want to add a disclaimer to the docs that if people do this, we can't guarantee that `z-index` and `overflow: hidden` will be accounted for correctly.

Reply:
Am I best putting that in this comment or in the .md file?

Reply:
I think it's better in the comment since it's right next to the code and will be easier to remember to update it if we decide to change the type.